src/io_scene_vrm/exporter/vrm_animation_exporter.py

```diff
@@ -770,22 +802,32 @@ def create_node_animation(
                 if index is None:
                     continue
                 euler_offset[index[fcurve.array_index]] = value
+
             elif property_name == "location":
-                if offset < len(hips_translation_offsets):
-                    translation_offset = hips_translation_offsets[offset]
+                # If it's hips
+                if bone.name == hips_bone_name:
+                    if offset < len(hips_translation_offsets):
+                        translation_offset = hips_translation_offsets[offset]
+                    else:
+                        translation_offset = Vector((0.0, 0.0, 0.0))
+                        hips_translation_offsets.append(translation_offset)
+                    translation_offset[fcurve.array_index] = value
+                # Else store for non-humanoid
                 else:
-                    translation_offset = Vector((0.0, 0.0, 0.0))
-                    hips_translation_offsets.append(translation_offset)
-                translation_offset[fcurve.array_index] = value
+                    loc_list = bone_name_to_location_offsets_for_non_humanoid[bone.name]
+                    if offset < len(loc_list):
+                        translation_offset = loc_list[offset]
+                    else:
+                        translation_offset = Vector((0.0, 0.0, 0.0))
+                        loc_list.append(translation_offset)
+                    translation_offset[fcurve.array_index] = value
 
     bone_name_to_quaternions: dict[str, list[Quaternion]] = {}
     for bone_name, quaternion_offsets in bone_name_to_quaternion_offsets.items():
         base_quaternion = bone_name_to_base_quaternion.get(bone_name)
         if base_quaternion is None:
             continue
         bone_name_to_quaternions[bone_name] = [
-            # ミュートされている項目とかあるとクオータニオンの値がノーマライズされて
-            # いないのでノーマライズしておく
```

Comment:
Please don't remove this comment. It's necessary. If you want to avoid Japanese, please send a translation pull request instead.